lib/Text_Wiki/Text/Wiki/Parse/Default/Tt.php

```diff
@@ -53,7 +53,11 @@ class Text_Wiki_Parse_Tt extends Text_Wiki_Parse {
     *
     */
 
-    public $regex = "/{{({*?.*}*?)}}/U";
+    public $regex = "/
+        {{
+        ({*?.*?}*?)  # Allows }, then text, then }. Sorry, what?
```

Comment:
```suggestion
        (\{*?.*?\}*?)  # Allows }, then text, then }. Sorry, what?
```

Reply:
Curly braces aren't a regex-active character in this context - if they were, the double braces on the previous and next lines would also need to be escaped. I really don't know what this regex is going for - the author clearly knows how to do "cannot start or end with X", as demonstrated elsewhere, so it's not that.